opennms-base-assembly/src/main/filtered/etc/events/opennms.events.xml

```diff
@@ -2078,4 +2078,26 @@
       <update-field field-name="severity" update-on-reduction="true"/>
     </alarm-data>
   </event>
+  
+   <event>
+    <uei>uei.opennms.org/internal/monitoringSystemAdded</uei>
+    <event-label>Monitoring system Added</event-label>
+    <descr> New monitoring system has been added </descr>
+    <logmsg dest="logndisplay">New monitoring System of type '%parm[monitoringSystemType]%' has been added with Id : %parm[monitoringSystemId]% at location '%parm[monitoringSystemLocation]%' </logmsg>
```

Comment:
- Change 'New ...' to 'A new...' to make this a complete sentence.
- Change 'Id' to 'ID'
- Lowercase 'System' since it is part of the sentence.
- Put a period at the end.
- Put single quotes (&quot;) around all of the parms for consistency.